autopts/ptsprojects/zephyr/bap.py

```diff
@@ -25,6 +25,8 @@
 from autopts.utils import ResultWithFlag
 
 broadcast_code = '0102680553F1415AA265BBAFC6EA03B8'
+broadcast_id = 0x123456
+broadcast_id_2 = 0x234567
```

Comment:
I know this is following the existing format, but generally in python constants are always in capital letters
```suggestion
BROADCAST_ID = 0x123456
BROADCAST_ID_2 = 0x234567
```